samples/sqltransport/multi-instance/SqlServer_3/Receiver/Program.cs

```diff
@@ -38,4 +39,22 @@ await endpointInstance.Stop()
             .ConfigureAwait(false);
     }
 
+    static class ConnectionProvider
+    {
+        const string DefaultConnectionString = @"Data Source=.\SqlExpress;Database=ReceiverCatalog;Integrated Security=True";
+        const string SenderConnectionString = @"Data Source=.\SqlExpress;Database=SenderCatalog;Integrated Security=True";
+
+        public static async Task<SqlConnection> GetConnection(string transportAddress)
+        {
+            var connectionString = transportAddress.StartsWith("Samples.SqlServer.MultiInstanceSender")
```

Comment:
Shouldn't the check be  `Samples.SqlServer.MultiInstanceReceiver` instead of `Samples.SqlServer.MultiInstanceSender` as this is in the `Receiver/Program.cs` ?

Wouldn't the transport address be MultiInstanceReceiver in this project?